python/tvm/contrib/hexagon/meta_schedule.py

```diff
@@ -100,7 +101,7 @@ def run(self, runner_inputs: List[RunnerInput]) -> List[RunnerFuture]:
 
 
 def _worker_func(hexagon_launcher, evaluator_config, alloc_repeat, artifact_path, args_info):
-    with hexagon_launcher.start_session() as session:
+    with create_session(hexagon_launcher._workspace, hexagon_launcher._rpc_info) as session:
```

Comment:
Why is `meta_schedule` accessing internal objects owned by the `hexagon_launcher`?

Reply:
`meta_schedule` is just creating a session and for that, we need to pass workspace and rpc_info from launcher to the session . I can make add `get` functions to HexagonLauncher instead of accessing them directory. wdyt?